### DISCUSSION TITLE
I'm facing a build issue in the Vercel CI/CD which I'm not getting in the Local Build

### DISCUSSION THREAD

**Discussion by Srijan-Baniyal** [vercel/community#1678] [Help]
### Summary

I need help in the building process of my app in the Vercel  CI/CD or CLI ... 

### Additional information

![image](https://user-images.githubusercontent.com/107460791/222665240-ffee0b40-774e-4c47-8452-ff890c617ed9.png)

This is the issue I'm facing in the Vercel CI/CD but not in the LOCAL build process

**Comment by amyegan** [comment]
Hi @Srijan-Baniyal! Do you have a public repo or a minimal reproduction that we can use to debug this?

**Reply by Srijan-Baniyal** [reply]
Yes, I do have a public repo called BlogIva .

You can use it for your debugging purposes.

**Comment by Srijan-Baniyal** [comment]
Now there is another issue I have solved this one. Now you can close this discussion if you want to
